.github/workflows/lintdiff-test.yaml

```diff
@@ -0,0 +1,47 @@
+name: LintDiff - Test
```

Comment:
You should be able to extend this template:

https://github.com/Azure/azure-rest-api-specs/blob/main/.github/workflows/_reusable-eng-tools-test.yaml

Reply:
Fixed in 4212529777e329e6481dd648baeaad430cbfa9d4